### PR TITLE
Update of AG Tracking Protection filter

### PR DESCRIPTION
**Pull request by ghost**
2ef0eb5c9d8f22c4cd02379a868f8afffde94cc5
https://gall.dcinside.com/board/view/?id=superidea&no=240566

ce34a53ea616d1eb2ce5490ec255a2a924234821
https://www.ruliweb.com/
https://bbs.ruliweb.com/hobby/board/300117/read/30627941
https://bbs.ruliweb.com/news/read/146836

a5dd3a09e035902b515da6e086d743aaddbe1ff2
https://www.ytn.co.kr/_ln/0103_202101312154352147

00390cbe4b65dc2922c9765c0d2e738fb62b19d3
https://www.clien.net/service/board/news/15852060
https://www.clien.net/service/

cc970c399ca45dc96c91184ca6f4bd2e90f40ff8, 3428799a0d926225785c9c0e914a733fa8ad4020
https://realty.chosun.com

8c9319537f2ef52975155d320d8acd61117c3f03
https://pann.nate.com/talk/357550580

a3e20e93735801ba0538174aff4b22685b0594f6
https://biz.heraldcorp.com/view.php?ud=20210131000291

03d686b151b7c9f3dd4c2fa98e1dd03d62557504
https://theqoo.net/index.php?mid=stock&document_srl=1853956740

### CODE REVIEW

**Comment by Yuki2718** [comment]
More of a matter of preference, but I personally prefer simplifying rules such that:
```
||auction.co.kr/cpc/imp/nw?r
||tend-table.com/cgi-bin/WebLog.dll
||ebaykorea.com/api/getImps
||zum.com/logs/ad/
||theqoo.net/counter.js
||clien.net/*/clien.count.js
||ruliweb.com/view/log.htm
||jennifersoft.com/api/collect-log
||chosun.com/dhtm/*/collecter.js
```
I think the shortened rules are still unique enough, reducing memory usage a tiny bit.

**Comment by piquark6046** [comment]
a41c09d7af0290a4b3f2fea152a244df1e54f2dc
https://github.com/AdguardTeam/AdguardFilters/issues/75539

**Comment by piquark6046** [comment]
40d6cc1d0f001759e5b64514099b59f8032a6a7f
https://w.atwiki.jp/senjounokizuna/

**Comment by Alex-302** [comment]
Please resolve one issue only in one pull request. Otherwise review process will be endless.

File: SpywareFilter/sections/specific.txt [hunk 1]
```diff
@@ -23,6 +23,18 @@
 ||adc.nine.com.au/?token=$redirect=nooptext,important,domain=9now.com.au
 ||googleadservices.com/pagead/conversion_async.js$redirect=noopjs,domain=ffm.to,important
 !
+||w.atwiki.jp/*/google_analytics.js
+||w.atwiki.jp/*/counter.js
+||sentry.afreecatv.com/api/5/store/?sentry_key=
```
Comment (Alex-302) [inline SpywareFilter/sections/specific.txt:28]: sentry is not a tracker. Used by developers for bug tracking.

File: SpywareFilter/sections/tracking_servers.txt [hunk 2]
```diff
@@ -42,6 +42,16 @@
 ||visilabs.com^$third-party
 ||visilabs.net^$third-party
 !
+||adtg.widerplanet.com^
+||adx.turl.co.kr^
+||ssp.realclick.co.kr^
+||kitweb.tadapi.info^
+||mediacategory.com^$third-party
+||adimg.wisenut.co.kr^
```
Comment (Alex-302) [inline SpywareFilter/sections/tracking_servers.txt:50]: ads are loaded from this domain?
I don't see all these requests on wisenut.co.kr

File: SpywareFilter/sections/tracking_servers_firstparty.txt [hunk 3]
```diff
@@ -26,9 +26,12 @@
 ! Disabled in DNS filter
 ||telemetry.mozilla.org^
 !
+||eventapi.afreecatv.com^
 ||stats.muasamcong.gov.vn^
 ||beacon.watch.impress.co.jp^
 ||media.fc2.com^$image
+||stat.nate.com^
```
Comment (Alex-302) [inline SpywareFilter/sections/tracking_servers_firstparty.txt:33]: also statetc.nate.com

File: SpywareFilter/sections/tracking_servers_firstparty.txt [hunk 4]
```diff
@@ -26,9 +26,12 @@
...
+||hitweb2.chosun.com^
```
Comment (Alex-302) [inline SpywareFilter/sections/tracking_servers_firstparty.txt:34]: not reproduced on chosun.com

**Review by Alex-302 — CHANGES_REQUESTED** [review]

File: SpywareFilter/sections/tracking_servers_firstparty.txt (hunk 3 above)
Comment (piquark6046) [reply SpywareFilter/sections/tracking_servers_firstparty.txt:33]: Screenshot
https://pann.nate.com/talk/357550580

File: SpywareFilter/sections/tracking_servers_firstparty.txt (hunk 4 above)
Comment (piquark6046) [reply SpywareFilter/sections/tracking_servers_firstparty.txt:34]: Screenshot
Screenshot
https://realty.chosun.com/
https://realty.chosun.com/site/data/html_dir/2021/02/24/2021022402779.html

**Comment by piquark6046** [comment]
1e9fed194c3311a86bd5531dc291fcf695a79a4b
https://realty.chosun.com/site/data/html_dir/2021/02/24/2021022402779.html

File: SpywareFilter/sections/tracking_servers.txt (hunk 2 above)
Comment (piquark6046) [reply SpywareFilter/sections/tracking_servers.txt]: https://github.com/AdguardTeam/AdguardFilters/pull/75313/commits/a2f01aa318abfc99817e47ae2cbe80c7193bfee9

File: SpywareFilter/sections/specific.txt (hunk 1 above)
Comment (piquark6046) [reply SpywareFilter/sections/specific.txt]: https://github.com/AdguardTeam/AdguardFilters/pull/75313/commits/cd7fb346d60350249e908a5f103df5d8f5b7d17d

**Comment by Alex-302** [comment]
Where `wisenut.co.kr`'s subdomains used? I don't see them on wisenut.co.kr

**Comment by piquark6046** [comment]
@Alex-302 Removed.
https://github.com/AdguardTeam/AdguardFilters/pull/75313/commits/b2f7f00a8cefc0595bb6ed7def5a2ca82783676a

**Comment by Alex-302** [comment]
Please devide rules by sites(just in comment, not in PR). This is impossible to check which tracker is first or trird-party, and to wtich site related.

**Comment by piquark6046** [comment]
> 
> 
> Please devide rules by sites(just in comment, not in PR). This is impossible to check which tracker is first or trird-party, and to wtich site related.

Did you mean what I close this issue, and open each pull request?

**Comment by Alex-302** [comment]
No. Just write a comment with all rules, grouped by sites.
Or, if you want - new PRs:)

**Comment by piquark6046** [comment]
@Alex-302 Is it okay?

**Comment by Alex-302** [comment]
I mean just message in our conversation - the list of rules with site names:) Comments near rules in commits are redundant.

File: SpywareFilter/sections/specific.txt [hunk 5]
```diff
@@ -23,6 +23,26 @@
 ||adc.nine.com.au/?token=$redirect=nooptext,important,domain=9now.com.au
 ||googleadservices.com/pagead/conversion_async.js$redirect=noopjs,domain=ffm.to,important
 !
+! realty.chosun.com - https://github.com/AdguardTeam/AdguardFilters/pull/75313
+||news.chosun.com/svc/fb_count.html
+||news.chosun.com/dhtm/*/collecter.js
+! w.atwiki.jp - https://github.com/AdguardTeam/AdguardFilters/pull/75313
+||w.atwiki.jp/*/google_analytics.js
```
Comment (Alex-302) [inline SpywareFilter/sections/specific.txt:30]: covered by `_analytics.js?`

File: SpywareFilter/sections/specific.txt [hunk 6]
```diff
@@ -23,6 +23,26 @@
...
+||w.atwiki.jp/*/counter.js
```
Comment (Alex-302) [inline SpywareFilter/sections/specific.txt:31]: covered by `/js/counter.js`

File: SpywareFilter/sections/specific.txt [hunk 7]
```diff
@@ -23,6 +23,26 @@
...
+! ytn.co.kr - https://github.com/AdguardTeam/AdguardFilters/pull/75313
+||ats.auction.co.kr/cpc/imp/nw?r
+||engine.tend-table.com/cgi-bin/WebLog.dll
+||grenade.ebaykorea.com/api/getImps
```
Comment (Alex-302) [inline SpywareFilter/sections/specific.txt:35]: Not reproduced:
```
||ats.auction.co.kr/cpc/imp/nw?r
||engine.tend-table.com/cgi-bin/WebLog.dll
||grenade.ebaykorea.com/api/getImps
```

File: SpywareFilter/sections/specific.txt [hunk 8]
```diff
@@ -23,6 +23,26 @@
...
+||openapi.tv.zum.com/logs/ad/
+! theqoo.net - https://github.com/AdguardTeam/AdguardFilters/pull/75313
+||cdn.theqoo.net/counter.js
+! clien.net - https://github.com/AdguardTeam/AdguardFilters/pull/75313
+||clien.net/service/*/clien.count.js
```
Comment (Alex-302) [inline SpywareFilter/sections/specific.txt:40]: image
they does not collect interesting info.

File: SpywareFilter/sections/specific.txt [hunk 9]
```diff
@@ -23,6 +23,26 @@
...
+! ruliweb.com - https://github.com/AdguardTeam/AdguardFilters/pull/75313
+||image.ruliweb.com/view/log.htm
```
Comment (Alex-302) [inline SpywareFilter/sections/specific.txt:42]: cannot reproduce

File: SpywareFilter/sections/specific.txt [hunk 10]
```diff
@@ -23,6 +23,26 @@
...
+! gall.dcinside.com - https://github.com/AdguardTeam/AdguardFilters/pull/75313
+||d-api.jennifersoft.com/api/collect-log
```
Comment (Alex-302) [inline SpywareFilter/sections/specific.txt:44]: Cannot open
image
It seems that service is for developers
http://cdn.jennifersoft.com/wp-content/uploads/Documents/en/JENNIFER_White_Paper-Performance_Analysis_Model.pdf

File: SpywareFilter/sections/tracking_servers.txt [hunk 11]
```diff
@@ -42,6 +42,11 @@
 ||visilabs.com^$third-party
 ||visilabs.net^$third-party
 !
+! pann.nate.com - https://github.com/AdguardTeam/AdguardFilters/pull/75313
+||adtg.widerplanet.com^
```
Comment (Alex-302) [inline SpywareFilter/sections/tracking_servers.txt:46]: not reproduced

**Review by Alex-302 — CHANGES_REQUESTED** [review]

File: SpywareFilter/sections/tracking_servers.txt (hunk 11 above)
Comment (piquark6046) [reply SpywareFilter/sections/tracking_servers.txt]: https://github.com/AdguardTeam/AdguardFilters/pull/75313/commits/b77b7616a50c637661f561a74c6208eeffc7d547

File: SpywareFilter/sections/specific.txt (hunk 10 above)
Comment (piquark6046) [reply SpywareFilter/sections/specific.txt]: https://github.com/AdguardTeam/AdguardFilters/pull/75313/commits/b77b7616a50c637661f561a74c6208eeffc7d547

File: SpywareFilter/sections/specific.txt (hunk 9 above)
Comment (piquark6046) [reply SpywareFilter/sections/specific.txt]: https://github.com/AdguardTeam/AdguardFilters/pull/75313/commits/b77b7616a50c637661f561a74c6208eeffc7d547

File: SpywareFilter/sections/specific.txt (hunk 8 above)
Comment (piquark6046) [reply SpywareFilter/sections/specific.txt]: https://github.com/AdguardTeam/AdguardFilters/pull/75313/commits/b77b7616a50c637661f561a74c6208eeffc7d547

File: SpywareFilter/sections/specific.txt (hunk 7 above)
Comment (piquark6046) [reply SpywareFilter/sections/specific.txt]: https://github.com/AdguardTeam/AdguardFilters/pull/75313/commits/b77b7616a50c637661f561a74c6208eeffc7d547

File: SpywareFilter/sections/specific.txt (hunk 6 above)
Comment (piquark6046) [reply SpywareFilter/sections/specific.txt]: https://github.com/AdguardTeam/AdguardFilters/pull/75313/commits/b77b7616a50c637661f561a74c6208eeffc7d547

File: SpywareFilter/sections/specific.txt (hunk 5 above)
Comment (piquark6046) [reply SpywareFilter/sections/specific.txt]: https://github.com/AdguardTeam/AdguardFilters/pull/75313/commits/b77b7616a50c637661f561a74c6208eeffc7d547

**Review by Alex-302 — APPROVED** [review]

**Comment by Alex-302** [comment]
Thanks.

**Pull request merged**